spec/controllers/medicaid/success_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe Medicaid::SuccessController do
+  before do
+    session[:medicaid_application_id] = current_app.id
+  end
+
+  describe "#edit" do
+    it "emails the medicaid application to the office" do
+      allow(ExportFactory).to receive(:create)
+
+      get :edit
+
+      expect(ExportFactory).to have_received(:create).
+        with(benefit_application: current_app, destination: :office_email)
+    end
+
+    context "application is not exportable" do
```

Comment:
Why would the application not be exportable?

Reply:
if it is not signed it is not exportable.

Reply:
How could you get to this step if that was the case? Especially if this is called the "success" controller.

Reply:
technically you shouldnt be able to...I am not sure what the historical reason for having this as a method was, but it is the pattern we had in SNAP so I figured we should replicate here. Perhaps we should page product/design on this?

Reply:
we removed this functionality!

Reply:
YAY LESS LOGIC :)